src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/UserController.kt

```diff
@@ -0,0 +1,26 @@
+package org.vaccineimpact.api.app.controllers
+
+import org.vaccineimpact.api.app.ActionContext
+import org.vaccineimpact.api.app.controllers.endpoints.SecuredEndpoint
+import org.vaccineimpact.api.app.errors.UnknownObjectError
+import org.vaccineimpact.api.models.User
+
+class UserController(context: ControllerContext) : AbstractController(context)
+{
+    override val urlComponent = "/users"
+    override val endpoints = listOf(
+            SecuredEndpoint("/:username/", this::getUser, emptySet())
+    )
+
+    fun getUser(context: ActionContext): User
+    {
+        var userName = userName(context)
+        var user = repos.user().use { it.getUserByUsername(userName(context))  }
+        if (user == null)
+            throw UnknownObjectError(userName, "Username")
```

Comment:
That makes a lot of sense. I'll leave `getUserByEmail` as is, agree with not using exceptions to handle flow, but I'll rename to `getMontaguUserByEmail` just to emphasize the different return type.